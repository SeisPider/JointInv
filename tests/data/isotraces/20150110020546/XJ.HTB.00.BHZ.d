SURF96 R U X -1 90.00000 3.37613 0.17673
SURF96 R U X -1 75.00000 3.85190 0.19171
SURF96 R U X -1 70.00000 3.79087 0.17330
SURF96 R U X -1 65.00000 3.71804 0.15480
SURF96 R U X -1 60.00000 3.79050 0.14852
SURF96 R U X -1 55.00000 3.73402 0.13211
SURF96 R U X -1 50.00000 3.70261 0.11809
SURF96 R U X -1 48.00000 3.66470 0.11106
SURF96 R U X -1 46.00000 3.59839 0.10261
SURF96 R U X -1 44.00000 3.55192 0.09563
SURF96 R U X -1 44.00000 4.08488 0.12649
SURF96 R U X -1 42.00000 3.54735 0.09105
SURF96 R U X -1 40.00000 3.54341 0.08652
SURF96 R U X -1 38.00000 3.50675 0.08051
SURF96 R U X -1 36.00000 3.43471 0.07317
SURF96 R U X -1 34.00000 3.39000 0.06731
SURF96 R U X -1 32.00000 3.36559 0.06245
SURF96 R U X -1 30.00000 3.31987 0.05696
SURF96 R U X -1 29.00000 3.29806 0.05434
SURF96 R U X -1 28.00000 3.28357 0.05201
SURF96 R U X -1 27.00000 3.26984 0.04973
SURF96 R U X -1 26.00000 3.25087 0.04734
SURF96 R U X -1 25.00000 3.23359 0.04503
