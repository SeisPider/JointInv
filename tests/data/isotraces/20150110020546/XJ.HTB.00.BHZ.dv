MFT96 R U T -1 91.022    3.6157    1.0000    5804.5     0.2662E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 89.043    3.6280    1.0000    5804.5     0.2985E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 87.149    3.6396    1.0000    5804.5     0.3414E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 85.333    3.6506    1.0000    5804.5     0.3970E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 83.592    3.6610    1.0000    5804.5     0.4657E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 81.920    3.6708    1.0000    5804.5     0.5467E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 80.314    3.6800    1.0000    5804.5     0.6387E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 78.769    3.6885    1.0000    5804.5     0.7400E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 77.283    3.6965    1.0000    5804.5     0.8487E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 75.852    3.7038    1.0000    5804.5     0.9632E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 74.473    3.7106    1.0000    5804.5     0.1082E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 73.143    3.7167    1.0000    5804.5     0.1204E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 71.860    3.7223    1.0000    5804.5     0.1329E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 70.621    3.7273    1.0000    5804.5     0.1456E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 69.424    3.7316    1.0000    5804.5     0.1586E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 68.267    3.7355    1.0000    5804.5     0.1719E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 67.148    3.7387    1.0000    5804.5     0.1855E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 66.065    3.7414    1.0000    5804.5     0.1996E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 65.016    3.7436    1.0000    5804.5     0.2142E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 64.000    3.7452    1.0000    5804.5     0.2294E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 63.015    3.7464    1.0000    5804.5     0.2453E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 62.061    3.7470    1.0000    5804.5     0.2620E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 61.134    3.7471    1.0000    5804.5     0.2794E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 60.235    3.7467    1.0000    5804.5     0.2976E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 59.362    3.7458    1.0000    5804.5     0.3165E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 58.514    3.7445    1.0000    5804.5     0.3363E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 57.690    3.7428    1.0000    5804.5     0.3567E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 56.889    3.7406    1.0000    5804.5     0.3777E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 56.110    3.7379    1.0000    5804.5     0.3992E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 55.351    3.7349    1.0000    5804.5     0.4211E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 54.613    3.7315    1.0000    5804.5     0.4432E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 53.895    3.7277    1.0000    5804.5     0.4655E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 53.195    3.7235    1.0000    5804.5     0.4878E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 52.513    3.7190    1.0000    5804.5     0.5101E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 51.848    3.7141    1.0000    5804.5     0.5324E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 51.200    3.7090    1.0000    5804.5     0.5549E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 50.568    3.7035    1.0000    5804.5     0.5777E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 49.951    3.6977    1.0000    5804.5     0.6013E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 49.349    3.6916    1.0000    5804.5     0.6262E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 48.762    3.6853    1.0000    5804.5     0.6531E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 48.188    3.6787    1.0000    5804.5     0.6826E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 47.628    3.6719    1.0000    5804.5     0.7156E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 47.080    3.6649    1.0000    5804.5     0.7528E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 46.545    3.6577    1.0000    5804.5     0.7948E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 46.022    3.6502    1.0000    5804.5     0.8420E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 45.511    3.6426    1.0000    5804.5     0.8945E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 45.011    3.6349    1.0000    5804.5     0.9523E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 44.522    3.6269    1.0000    5804.5     0.1015E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 44.043    3.6189    1.0000    5804.5     0.1081E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 43.574    3.6107    1.0000    5804.5     0.1151E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 43.116    3.6024    1.0000    5804.5     0.1222E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 42.667    3.5940    1.0000    5804.5     0.1294E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 42.227    3.5855    1.0000    5804.5     0.1364E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 41.796    3.5770    1.0000    5804.5     0.1433E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 41.374    3.5684    1.0000    5804.5     0.1497E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 40.960    3.5597    1.0000    5804.5     0.1557E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 40.554    3.5510    1.0000    5804.5     0.1611E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 40.157    3.5423    1.0000    5804.5     0.1658E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 39.767    3.5336    1.0000    5804.5     0.1698E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 39.385    3.5248    1.0000    5804.5     0.1731E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 39.010    3.5161    1.0000    5804.5     0.1756E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 38.642    3.5074    1.0000    5804.5     0.1774E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 38.280    3.4987    1.0000    5804.5     0.1785E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 37.926    3.4900    1.0000    5804.5     0.1791E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 37.578    3.4814    1.0000    5804.5     0.1793E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 37.236    3.4729    1.0000    5804.5     0.1792E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 36.901    3.4644    1.0000    5804.5     0.1790E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 36.571    3.4561    1.0000    5804.5     0.1788E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 36.248    3.4478    1.0000    5804.5     0.1788E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 35.930    3.4396    1.0000    5804.5     0.1792E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 35.617    3.4315    1.0000    5804.5     0.1800E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 35.310    3.4235    1.0000    5804.5     0.1815E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 35.009    3.4157    1.0000    5804.5     0.1836E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 34.712    3.4079    1.0000    5804.5     0.1863E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 34.420    3.4004    1.0000    5804.5     0.1898E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 34.133    3.3929    1.0000    5804.5     0.1940E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 33.851    3.3857    1.0000    5804.5     0.1987E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 33.574    3.3786    1.0000    5804.5     0.2038E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 33.301    3.3716    1.0000    5804.5     0.2093E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 33.032    3.3649    1.0000    5804.5     0.2150E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 32.768    3.3583    1.0000    5804.5     0.2207E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 32.508    3.3520    1.0000    5804.5     0.2263E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 32.252    3.3458    1.0000    5804.5     0.2316E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 32.000    3.3399    1.0000    5804.5     0.2366E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 31.752    3.3342    1.0000    5804.5     0.2411E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 31.508    3.3287    1.0000    5804.5     0.2452E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 31.267    3.3234    1.0000    5804.5     0.2488E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 31.030    3.3184    1.0000    5804.5     0.2519E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 30.797    3.3136    1.0000    5804.5     0.2547E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 30.567    3.3091    1.0000    5804.5     0.2571E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 30.341    3.3048    1.0000    5804.5     0.2594E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 30.118    3.3008    1.0000    5804.5     0.2617E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 29.898    3.2971    1.0000    5804.5     0.2641E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 29.681    3.2937    1.0000    5804.5     0.2668E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 29.468    3.2906    1.0000    5804.5     0.2699E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 29.257    3.2878    1.0000    5804.5     0.2736E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 29.050    3.2852    1.0000    5804.5     0.2779E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 28.845    3.2830    1.0000    5804.5     0.2829E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 28.643    3.2812    1.0000    5804.5     0.2887E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 28.444    3.2796    1.0000    5804.5     0.2953E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 28.248    3.2784    1.0000    5804.5     0.3026E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 28.055    3.2776    1.0000    5804.5     0.3105E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 27.864    3.2771    1.0000    5804.5     0.3188E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 27.676    3.2770    1.0000    5804.5     0.3274E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 27.490    3.2772    1.0000    5804.5     0.3360E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 27.307    3.2778    1.0000    5804.5     0.3442E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 27.126    3.2789    1.0000    5804.5     0.3519E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 26.947    3.2803    1.0000    5804.5     0.3584E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 26.771    3.2822    1.0000    5804.5     0.3636E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 26.597    3.2844    1.0000    5804.5     0.3671E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 26.426    3.2872    1.0000    5804.5     0.3685E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 26.256    3.2903    1.0000    5804.5     0.3676E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 26.089    3.2939    1.0000    5804.5     0.3641E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 25.924    3.2980    1.0000    5804.5     0.3579E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 25.761    3.3026    1.0000    5804.5     0.3490E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 25.600    3.3077    1.0000    5804.5     0.3372E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 25.441    3.3133    1.0000    5804.5     0.3229E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 25.284    3.3194    1.0000    5804.5     0.3061E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 25.129    3.3261    1.0000    5804.5     0.2871E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 90.000    3.6220    1.0000    5804.5     0.2829E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 85.000    3.6526    1.0000    5804.5     0.4101E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 80.000    3.6817    1.0000    5804.5     0.6593E-06   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 75.000    3.7080    1.0000    5804.5     0.1037E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 70.000    3.7295    1.0000    5804.5     0.1524E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 65.000    3.7436    1.0000    5804.5     0.2144E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 60.000    3.7465    1.0000    5804.5     0.3027E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 55.000    3.7333    1.0000    5804.5     0.4316E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 50.000    3.6981    1.0000    5804.5     0.5994E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 48.000    3.6764    1.0000    5804.5     0.6937E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 46.000    3.6499    1.0000    5804.5     0.8443E-05   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 44.000    3.6181    1.0000    5804.5     0.1088E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 42.000    3.5810    1.0000    5804.5     0.1400E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 40.000    3.5388    1.0000    5804.5     0.1674E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 38.000    3.4918    1.0000    5804.5     0.1790E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 36.000    3.4414    1.0000    5804.5     0.1791E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 34.000    3.3895    1.0000    5804.5     0.1962E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 32.000    3.3399    1.0000    5804.5     0.2366E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 30.000    3.2989    1.0000    5804.5     0.2630E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 29.000    3.2847    1.0000    5804.5     0.2791E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 28.000    3.2774    1.0000    5804.5     0.3128E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 27.000    3.2799    1.0000    5804.5     0.3565E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 26.000    3.2962    1.0000    5804.5     0.3608E-04   -5.655000   68.361298   44.086899   86.584396 0  1 COMMENT
