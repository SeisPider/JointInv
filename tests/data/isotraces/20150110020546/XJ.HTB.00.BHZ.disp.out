SURF96 R U T -1 90.000    3.6220    1.0000    
SURF96 R C T -1 90.000    6.3671    1.0000    
SURF96 R U T -1 85.000    3.6526    1.0000    
SURF96 R C T -1 85.000    6.1121    1.0000    
SURF96 R U T -1 80.000    3.6817    1.0000    
SURF96 R C T -1 80.000    5.8815    1.0000    
SURF96 R U T -1 75.000    3.7080    1.0000    
SURF96 R C T -1 75.000    5.6718    1.0000    
SURF96 R U T -1 70.000    3.7295    1.0000    
SURF96 R C T -1 70.000    5.4800    1.0000    
SURF96 R U T -1 65.000    3.7436    1.0000    
SURF96 R C T -1 65.000    5.3033    1.0000    
SURF96 R U T -1 60.000    3.7465    1.0000    
SURF96 R C T -1 60.000    5.1390    1.0000    
SURF96 R U T -1 55.000    3.7333    1.0000    
SURF96 R C T -1 55.000    4.9839    1.0000    
SURF96 R U T -1 50.000    3.6981    1.0000    
SURF96 R C T -1 50.000    4.8341    1.0000    
SURF96 R U T -1 48.000    3.6764    1.0000    
SURF96 R C T -1 48.000    4.7747    1.0000    
SURF96 R U T -1 46.000    3.6499    1.0000    
SURF96 R C T -1 46.000    4.7151    1.0000    
SURF96 R U T -1 44.000    3.6181    1.0000    
SURF96 R C T -1 44.000    4.6549    1.0000    
SURF96 R U T -1 42.000    3.5810    1.0000    
SURF96 R C T -1 42.000    4.5937    1.0000    
SURF96 R U T -1 40.000    3.5388    1.0000    
SURF96 R C T -1 40.000    4.5311    1.0000    
SURF96 R U T -1 38.000    3.4918    1.0000    
SURF96 R C T -1 38.000    4.4665    1.0000    
SURF96 R U T -1 36.000    3.4414    1.0000    
SURF96 R C T -1 36.000    4.3997    1.0000    
SURF96 R U T -1 34.000    3.3895    1.0000    
SURF96 R C T -1 34.000    4.3303    1.0000    
SURF96 R U T -1 32.000    3.3399    1.0000    
SURF96 R C T -1 32.000    4.2583    1.0000    
SURF96 R U T -1 30.000    3.2989    1.0000    
SURF96 R C T -1 30.000    4.1842    1.0000    
SURF96 R U T -1 29.000    3.2847    1.0000    
SURF96 R C T -1 29.000    4.1467    1.0000    
SURF96 R U T -1 28.000    3.2774    1.0000    
SURF96 R C T -1 28.000    4.1093    1.0000    
SURF96 R U T -1 27.000    3.2799    1.0000    
SURF96 R C T -1 27.000    4.0724    1.0000    
SURF96 R U T -1 26.000    3.2962    1.0000    
SURF96 R C T -1 26.000    4.0366    1.0000    
